cartoframes/data/dataset/registry/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import
```

Comment:
I think we can remove this if we are not using and import in the file

Reply:
Honestly I'm not 100% sure of that cause I think Sphinx needs it to use `automodule` (for Pyton 2.7), but I'll test it